src/main/java/Duke/command/Command.java

```diff
@@ -0,0 +1,8 @@
+package Duke.command;
+import Duke.*;
+import Duke.task.*;
+public abstract class Command{
+    public abstract void execute(TaskList tsk, Ui ui, Storage storage);
```

Comment:
Perhaps you could standardise your naming of tsk, with 's' or no 's' throughout all your overridden methods, to help with readability